src/Symfony/Component/Serializer/Tests/Fixtures/Annotations/IgnoreDummyAdditionalGetter.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
This should be removed in all classes as we're not using this in Symfony.

Reply:
I was not aware of that. Is now corrected. I also removed it from two entities that I had added in my last bug fix with the same problem.

Reply:
Shouldn't this rule perhaps also be checked automatically? I couldn't find anything quickly but why don't you use strict_types?